lightcycle-integration-test/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,28 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.soundcloud.lightcycle.sample.basic">
```

Comment:
Agreed, at first I want to use the same package as the sample, but the code was growing so it'll be better if we change the package name